src/target/adiv5.c

```diff
@@ -216,6 +220,7 @@ static const struct {
 	{0xc09, aa_cortexa,   cidc_dc,      PIDR_PN_BIT_STRINGS("Cortex-A9 Debug", "(Debug Unit)")},
 	{0xc0f, aa_nosupport, cidc_unknown, PIDR_PN_BIT_STRINGS("Cortex-A15 Debug", "(Debug Unit)")}, /* support? */
 	{0xc14, aa_nosupport, cidc_unknown, PIDR_PN_BIT_STRINGS("Cortex-R4 Debug", "(Debug Unit)")}, /* support? */
+	{0xcd0, aa_nosupport, cidc_unknown, PIDR_PN_BIT_STRINGS("Atmel DSU", "(Device Service Unit)")},
```

Comment:
I added the DSU identifier here, to make adiv5_component_probe succeed when do_test = true. If not, cortexm_prepare is called here: https://github.com/UweBonnes/blackmagic/blob/3cac3d9cdb7e0dc17f75134cc7a674e81af94b92/src/target/adiv5.c#L514-L517
and as cortexm_prepare returns false, probing fails.